### ISSUE TITLE
BUG: PDF Preview displaying error

### ISSUE THREAD

**Issue by klohman** [ustaxcourt/ef-cms#7476]
**Describe the Bug**
PDF preview not displaying document; error displays. This occurs on Correspondence screen and message detail (when correspondence doc is attached)

**Business Impact/Reason for Severity**

**In which environment did you see this bug?**
flexion prod 

**Who were you logged in as?**
petitions clerk 2

**What were you doing when you discovered this bug? (Using the application, demoing, smoke tests, testing other functionality, etc.)**
manual testing 

**To Reproduce**
1. Go to case (101-20) 
2. Click correspondence tab

**Expected Behavior**
PDF preview should display document. 

**Actual Behavior**
PDF preview not displaying document; error displays.

**Screenshots**
![Screen Shot 2020-11-06 at 9.23.07 AM.png](https://images.zenhubusercontent.com/5bd7271508843955cc24d3d0/5a92733f-81fa-42a7-9e08-801ef909430a)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Cause of Bug, If Known**

**Process for Logging a Bug:**
* Complete the above information
* Add a severity tag (Critical, High Severity, Medium Severity or Low Severity). See below for priority definition. 

**Severity Definition:**
* Critical Defect
Blocks entire system's or module’s functionality
No workarounds available
Testing cannot proceed further without bug being fixed.

* High-severity Defect
Affects key functionality of an application
There's a workaround, but not obvious or easy
App behaves in a way that is strongly different from the one stated in the requirements

* Medium-severity Defect
A minor function does not behave in a way stated in the requirements.
Workaround is available and easy

* Low-severity Defect
Mostly related to an application’s UI
Doesn't need a workaround, because it doesn't impact functionality

**FOR ENGINEERING TEAM ONLY**
Bug Resolution Steps:
 - [ ] Determine why the bug wasn't caught by a test. 
 - [ ] Determine if an automated test needs to fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] Determine if a manual test needs to be fixed, expanded or created. If unsure, bring in others to discuss.
 - [ ] If needed, automated test is created.
 - [ ] If needed, manual test is created.
 - [ ] Reason for bug has been documented.
 - [ ] Fix has been deployed to dev environment.
 - [ ] Fix has been deployed to the stage environment.
 - [ ] Bug has been tested in staging (UX or Engineering).

**Comment by rachaelparris** [comment]
These are migrated correspondence docs that did not have associated PDFs uploaded. I uploaded a matching PDF for the `receipt` correspondence doc (on 101-20) to S3, and that PDF is showing up now in the UI. I also uploaded another correspondence doc with the title `another test` to verify that uploading a correspondence doc attaches the PDF correctly, and it seems to work fine. I'm not sure if there is anything we should/could do to fix this issue, but I believe it shouldn't be an issue on the court's environments because the PDFs will be migrated along with the data.
